cluster/gce/gci/configure-helper.sh

```diff
@@ -224,6 +224,9 @@ function create-master-auth {
   fi
   local -r basic_auth_csv="${auth_dir}/basic_auth.csv"
   if [[ -n "${KUBE_PASSWORD:-}" && -n "${KUBE_USER:-}" ]]; then
+    if [[ -e "${basic_auth_csv}" ]]; then
+      sed -i '/admin,system:masters$/d' "${basic_auth_csv}"
```

Comment:
do we expect the old credentials to stop working on restart? that would surprise me